addons/web/static/src/views/fields/attachment_image/attachment_image_field.xml

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
This commit remove the title on the attachment image widget (in normal mode). 
??? Would be more precise